runtime/src/snapshot_utils.rs

```diff
@@ -396,6 +402,11 @@ pub fn archive_snapshot_package(
     datapoint_info!(
         "archive-snapshot-package",
         ("slot", snapshot_package.slot(), i64),
+        (
+            "format",
+            snapshot_package.archive_format().to_string(),
+            String
```

Comment:
Too bad `datapoint_info!()` doesn't support `&str` at the moment, could've avoided an allocation here (but not a big deal in this case). No actions needed here.